java/src/main/java/io/cucumber/cienvironment/CiEnvironmentImpl.java

```diff
@@ -80,28 +78,27 @@ private Git detectGit(Map<String, String> env, Function<Path, Stream<String>> ge
         );
     }
 
-    private String evaluateRevision(Map<String, String> env, Function<Path, Stream<String>> getLines) {
+    private String evaluateRevision(Map<String, String> env) {
         if ("pull_request".equals(env.get("GITHUB_EVENT_NAME"))) {
             if (env.get("GITHUB_EVENT_PATH") == null) {
                 throw new RuntimeException("GITHUB_EVENT_PATH not set");
             }
             Path path = Paths.get(env.get("GITHUB_EVENT_PATH"));
-            return getJsonProperty(path, getLines, "after");
+            try {
+                String json = Files.lines(path).collect(Collectors.joining());
```

Comment:
Might be better to read all bytes instead. Less fragile.

Reply:
Yep done